test/onnx/test_fx_op_consistency.py

```diff
@@ -391,12 +391,12 @@
     ),
     xfail(
         "nn.functional.max_pool2d",
-        dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
```

Comment:
why `torch.int8` was removed (by expanding `onnx_test_common.INT_TYPES`)?

Reply:
Because it now passes the tests. I didn't dig in to why. I imagine it's because I've fixed our implementation